convert/README.md

```diff
@@ -0,0 +1,185 @@
+# Domino CDK to Terraform migration
+
+Domino is deprecating this CDK repository, and switching all future EKS
+provisioning work to our Terraform module, located here:
+
+https://github.com/dominodatalab/terraform-aws-eks
+
+The scripts in this directory will help you convert the management your
```

Comment:
I think this will be clearer without `the management`

Reply:
The reason I included that phrase is to try and reinforce the idea that this isn't any sort of lift-and-shift migration.